st2common/st2common/util/pack_management.py

```diff
@@ -151,7 +151,22 @@ def clone_repo(temp_dir, repo_url, verify_ssl=True, ref='master'):
     # because we want the user to work with the repo in the
     # future.
     repo = Repo.clone_from(repo_url, temp_dir)
-    active_branch = repo.active_branch
+
+    is_local_repo = repo_url.startswith('file:///')
```

Comment:
File URI can be with just 2 slashes in it like `file://localhost/some/local/dir`, see https://en.wikipedia.org/wiki/File_URI_scheme

Reply:
Yeah, I don't think rest of our pack management code code supports that though :)

Reply:
You are correct - we didn't have any tests cases so I assume it's not supported / doesn't work.

Will make a change and add a test case.